ide/idetop.ml

```diff
@@ -491,7 +491,10 @@ let msg_format = ref (fun () ->
 
 (* The loop ignores the command line arguments as the current model delegates
    its handing to the toplevel container. *)
-let loop ~opts:_ ~state =
+let loop ~opts:_ run_mode state =
+  match run_mode with
+  | Coqtop.Batch -> exit 0
+  | Coqtop.Interactive ->
```

Comment:
Why this change?

Reply:
The idea was to remove `-batch` from the common command line options parser to the executables for which it really matters. In particular, I could not see a meaning for `-batch` for `coqc`. So I dispatched the test for `Batch` vs `Interactive` mode in `coqtop` and `coqidetop`: `coqidetop` is still called both interactively (to communicate really) and batch (to check connection).

What did you have in mind?

Reply:
Thanks for the explanation, I understand now; I just didn't get the full picture.

Reply:
Maybe you would like to write a more descriptive commit name tho, "fix" seems too terse IMVHO.

Reply:
Sorry, I intended to squash it with the commit just before then forgot. Thanks for noticing. I now squashed it.

Reply:
Thanks @herbelin , will try to have a look and merge ASAP.